functions/offset.py

```diff
@@ -0,0 +1,41 @@
+def offset(proposed_mode,chosen_mode):
+   """
+   Function takes proposed mode as one argument which is a dictionary of modes of transport based on distance,
+   chosen mode as another argument which is a dictionary of one selected mode by user for that journey
+   :param proposed_mode:
+   :param chosen_mode:
+
+   """
+   prop_trans=max(proposed_mode, key=proposed_mode.get)
+   prop_emission_max= proposed_mode.get(prop_trans)
+   #print(prop_emission_max)
+   for key in chosen_mode:
+     chose_emission= chosen_mode.get(key)
+   #print(chose_emission)
+   def emission_return(prop_emission_max,chose_emission):
+       """
+       This function calculates offset(difference of carbon consumed/saved) and returns
+       offset of carbon consumed/saved for that journey
+       :param proposed emission max
+       :param chosen mode emission
+       :return offset
+        """
+       offset=prop_emission_max-chose_emission
+       if offset==0:
+           print("You have contributed ZERO carbon emission, NO EMISSION NO SAVING")
+       elif offset>0:

```

Comment:
Just as clarification as we shouldn't be printing in this function. If offset>0 this is the amount saved, and we can't ever get offset<0 as we are comparing to the max.